docs/content.zh/docs/ops/batch/batch_shuffle.md

```diff
@@ -114,12 +114,34 @@ Hybrid shuffle provides two spilling strategies:
 
 To use hybrid shuffle mode, you need to configure the [execution.batch-shuffle-mode]({{< ref "docs/deployment/config" >}}#execution-batch-shuffle-mode) to `ALL_EXCHANGES_HYBRID_FULL` (full spilling strategy) or `ALL_EXCHANGES_HYBRID_SELECTIVE` (selective spilling strategy).
 
+#### Supports AdaptiveBatchScheduler and SpeculativeExecution
+
+Hybrid shuffle currently supports `AdaptiveBatchScheduler` by default. If you want to use `DefaultScheduler`, please configure the [jobmanager.scheduler]({{< ref "docs/deployment/config" >}}#jobmanager-scheduler) to `DefaultScheduler`. See [elastic_scaling]({{< ref "docs/deployment/elastic_scaling" >}}#adaptive-batch-scheduler) for details.
+
+If you want to enable `SpeculativeExecution` in the same time, see [speculative_execution]({{< ref "docs/deployment/speculative_execution" >}}) for details.
+
+Hybrid shuffle divides the partition data consumption constraints between producer and consumer into the following three cases:
+
+- **ALL_PRODUCERS_FINISHED** : hybrid partition data can be consumed only when all producers are finished.
+- **ONLY_FINISHED_PRODUCERS** : hybrid partition data can be consumed when its producer is finished.
+- **UNFINISHED_PRODUCERS** : hybrid partition data can be consumed even if its producer is un-finished.
+
+If `SpeculativeExecution` is enabled, the default constraint is `ONLY_FINISHED_PRODUCERS` to bring some performance optimization compared with blocking shuffle. Otherwise, the default constraint is `UNFINISHED_PRODUCERS` to perform pipelined-like shuffle. These could be configured via [jobmanager.partition.hybrid.partition-data-consume-constraint]({{< ref "docs/deployment/config" >}}#jobmanager-partition-hybrid-partition-data-consume-constraint).
```

Comment:
What is the potential impacts when changing this option?

Reply:
Added some descriptions of this part.